plugins/outputs/cratedb/cratedb.go

```diff
@@ -171,12 +187,15 @@ func escapeObject(m map[string]interface{}) (string, error) {
 	// Now we build our key = val pairs
 	pairs := make([]string, 0, len(m))
 	for _, k := range keys {
-		// escape the value of our key k (potentially recursive)
+		key := strings.ReplaceAll(escapeString(k, `"`), ".", "_")
```

Comment:
Can we please make the "replacement character" configurable? Or is underscore the only valid replacement?